### PR TITLE
STACK_ as prefix; wrong case in dir; NAME instead of PROJECT

### PR DESCRIPTION
**Pull request by fredliporace**
:wave:

I personally prefer TITILER_ as prefix instead of STACK_, if you agree I can change the PR to use TITILER_. But as it is it makes things consistent with STACK_.

### CODE REVIEW

File: docs/deployment/aws/ecs.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ The example handles tasks such as generating a docker image and setting up an ap
     $ git clone https://github.com/developmentseed/titiler.git
 
     # install cdk dependencies
-    $ cd titiler/deployment/aws && pip install -r requirements.txt
+    $ cd titiler/deployment/AWS && pip install -r requirements.txt
```
Comment (kylebarron) [inline docs/deployment/aws/ecs.md:19]: Why this change? Isn't the directory lower case `aws`?

Comment (kylebarron) [reply docs/deployment/aws/ecs.md:19]: Ah I was on mobile and the app showed lower case `aws` in the file name. I'd strongly suggest making the directory lower case instead.

Comment (kylebarron) [reply docs/deployment/aws/ecs.md:19]: To expand a little more: I think Windows and Mac filesystems are case-insensitive, and Linux is case-sensitive. So in a previous project with users on multiple OSes, we found hard-to-debug issues due to case sensitivity. So I'd argue it's best practice to just always use lower case.

Comment (vincentsarago) [reply docs/deployment/aws/ecs.md:19]: made is lower case https://github.com/developmentseed/titiler/commit/c952bd445e545bc8390c413d5f6fb0b6258dd071

**Comment by vincentsarago** [comment]
@fredliporace thanks for this PR.

I think it will be good to use `TITILER_STACK` instead of STACK or TITILER. 
https://github.com/developmentseed/titiler/blob/c952bd445e545bc8390c413d5f6fb0b6258dd071/deployment/AWS/cdk/config.py#L71

we can also change the env of the API to TITILER_API (instead of API)
https://github.com/developmentseed/titiler/blob/c952bd445e545bc8390c413d5f6fb0b6258dd071/titiler/settings.py#L27

File: docs/deployment/aws/ecs.md (hunk 1 above)
Comment (vincentsarago) [reply docs/deployment/aws/ecs.md:19]: 😬 in fact I didn't fixed this yet!

Comment (vincentsarago) [reply docs/deployment/aws/ecs.md:19]: done https://github.com/developmentseed/titiler/commit/553ac43c4e026b70e26a189520db74f90c211c06 cc @fredliporace

**Comment by fredliporace** [comment]
@vincentsarago I was working on a branch based on 0.1.0a14 and commiting simple documentation updates I flagged while updating my own deployment. Changing to TITILER_API seems to affect some testing and in my environment the testing fails for some reason I have to investigate... This last commit seems to be cover all the needed changes but right now I'm unable to run all tests locally as I would like.

**Comment by vincentsarago** [comment]
@fredliporace the tests seems to work on CI 🤷 

Maybe I'll wait for you to 👍 before merging.

File: CHANGES.md [hunk 2]
```diff
@@ -5,6 +5,7 @@
 * update FastAPI requirements
 * add `validate` in `MosaicTilerFactory` (https://github.com/developmentseed/titiler/pull/206, author @drnextgis)
 * rename `ressources` package to `resources` (https://github.com/developmentseed/titiler/pull/210, author @drnextgis)
+* ```TITILER_STACK``` as prefix to CDK; ```TITILER_API``` as prefix to API (https://github.com/developmentseed/titiler/pull/211, author @fredliporace)
```
Comment (vincentsarago) [inline CHANGES.md:8]:
```suggestion
* renamed environment variables prefixes for API and STAC configurations: `TITILER_STACK` as prefix to CDK and `TITILER_API` as prefix to API (https://github.com/developmentseed/titiler/pull/211, author @fredliporace)
```

**Comment by fredliporace** [comment]
> @fredliporace the tests seems to work on CI 
> 
> Maybe I'll wait for you to  before merging.

I like to run the testing locally and on my repo CI before submitting the PR. Seems to be OK.

**Comment by vincentsarago** [comment]
@fredliporace run the test on my computer and everything looks 👌. I'll merge this great PR 🙏

File: CHANGES.md [hunk 3]
```diff
@@ -5,6 +5,7 @@
 * update FastAPI requirements
 * add `validate` in `MosaicTilerFactory` (https://github.com/developmentseed/titiler/pull/206, author @drnextgis)
 * rename `ressources` package to `resources` (https://github.com/developmentseed/titiler/pull/210, author @drnextgis)
+* renamed environment variables prefixes for API and STACK configurations: ````TITILER_STACK``` as prefix to CDK and ```TITILER_API``` as prefix to API (https://github.com/developmentseed/titiler/pull/211, author @fredliporace)
```
Comment (vincentsarago) [inline CHANGES.md:8]:
```suggestion
* renamed environment variables prefixes for API and STACK configurations: `TITILER_STACK` as prefix to CDK and `TITILER_API` as prefix to API (https://github.com/developmentseed/titiler/pull/211, author @fredliporace)
```
